src/main/java/hudson/plugins/git/opt/PreBuildMergeOptions.java

```diff
@@ -53,13 +53,13 @@ public void setMergeTarget(String mergeTarget) {
     @Exported
     public MergeCommand.Strategy getMergeStrategy() {
         for (MergeCommand.Strategy strategy: MergeCommand.Strategy.values())
-            if (strategy.toString().equals(mergeStrategy))
+            if (strategy.equals(mergeStrategy))
                 return strategy;
         return MergeCommand.Strategy.DEFAULT;
```

Comment:
As written, this is silly; should just read

```java
return mergeStrategy;
```

But if the field type is reverted, then the rest of the changes to this class should be reverted, too.